pkg/network/dhcp_daemon.go

```diff
@@ -6,6 +6,28 @@ import (
 	"log"
 )
 
+// rawConfigHasIPAM determines if the rawConfig JSON data has IPAM element.
+func rawConfigHasIPAM(rawConfig map[string]interface{}) bool {
```

Comment:
Hey Daein, just a nit, but can we call this something related to the fact that it has DHCP in the IPAM, such as `rawConfigIPAMHasDHCP`? Thanks!

Reply:
Good point, I've changed the function name as `rawConfigIPAMHasDHCP` as you mentioned. Thanks!